units/pod_allocator.go

```diff
@@ -231,6 +233,16 @@ func (j *podAllocatorJob) populate(ctx context.Context) error {
 		j.pRef = pRef
 	}
 
+	if j.expansions == nil {
```

Comment:
nit: if we don't plan to use `j.expansions` outside of this populate function, can we just make it a local variable like how it's done in `model/project_ref.go`? These expansions seem like they're meant for a pretty specific purpose (which is to expand the container image).

Or did you plan to test the expansion behavior and this field makes it easier?

Reply:
I did this as an attempt to group the vars fetching with all of the other DB operations this job uses in its `populate` function for organizational purposes, but since it's used in only one spot like you said I'm fine to change it to be a local variable.